horace_core/sqw/PixelData/@pix_metadata/private/set_alignment_matr_.m

```diff
@@ -1,5 +1,8 @@
 function  obj = set_alignment_matr_(obj,val)
 %SET_ALIGNMENT_MATR_ helper property which checks and sets alignment matrix
+% if alignment matrix differs from unary matrix, it is stored in the class
```

Comment:
unary --> unit

this is not quite correct; alignment_matr_ is always set below. What is added is a min threshold for the difference, otherwise the unit matrix is stored. There is no attempt to assume that e.g. alignment_matr_==[] will be expanded to eye on the fly